apps/server/src/app.module.ts

```diff
@@ -4,10 +4,12 @@ import { DataSource } from 'typeorm';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { AuthModule } from './auth/auth.module';
+import { UserModule } from './user/user.module';
 import databaseConfig from './config/database.config';
 
 @Module({
-  imports: [ConfigModule.forRoot(), TypeOrmModule.forRoot(databaseConfig())],
+  imports: [ConfigModule.forRoot(), TypeOrmModule.forRoot(databaseConfig()), AuthModule, UserModule],
```

Comment:
The name `databaseConfig()` for a function is very vauge and unclear. Maybe it could be called `getDBConfig()` or smth else.

And I'm not sure if we need a function to get the databaseConfig. It could just be a `databaseConfig` object with the config options.

Reply:
updated the name to `getDBConfig()` instead of `databaseConfig()`